cmd/zc_traverser_local.go

```diff
@@ -197,11 +197,23 @@ func WalkWithSymlinks(appCtx context.Context, fullPath string, walkFunc filepath
 				computedRelativePath = ""
 			}
 
+			if (fileInfo.Mode()&os.ModeSocket != 0) || (fileInfo.Mode()&os.ModeNamedPipe != 0) || (fileInfo.Mode()&os.ModeIrregular != 0) {
+				WarnStdoutAndScanningLog(fmt.Sprintf("Unsupported file type %s: %v", filePath, fileInfo.Mode()))
+				return nil
+			}
```

Comment:
nit,

// TODO: Later we might want to transfer these special files as such.
unsupportedFileTypes := (os.ModeSocket | os.ModeNamedPipe | os.ModeIrregular | os.ModeDevice)
if ((fileInfo.Mode() & unsupportedFileTypes) != 0) {
}